dashboard/lib/queries/user/expired_deleted_accounts.rb

```diff
@@ -0,0 +1,28 @@
+module Queries
+  module User
+    # Retrieve all users who have been soft-deleted for at least the specified duration. Defaults to 28 days,
+    # which is the current grace period before rendering accounts unrecoverable during the PII purge process.
+    # Does not include accounts that have already been scrubbed of PII or purged/anonymized. These accounts are
+    # "hard-deleted" from a functional perspective, and are present only for analytics and reporting purposes.
+    class ExpiredDeletedAccounts < Queries::Base
+      def initialize(deleted_before: ::User::Purgeable::SOFT_DELETED_USER_TTL.ago)
```

Comment:
Wouldn't it be better to add this in the `included` block and call it from the `User` namespace instead?
```suggestion
      def initialize(deleted_before: ::User::SOFT_DELETED_TTL.ago)
```